geoviews/element/geo.py

```diff
@@ -10,9 +10,9 @@
 from holoviews.core import util
 from holoviews.element import (
     Contours as HvContours, Graph as HvGraph, Image as HvImage,
-    Nodes as HvNodes, Path as HvPath, Polygons as HvPolygons,
-    RGB as HvRGB, Text as HvText, TriMesh as HvTriMesh,
-    QuadMesh as HvQuadMesh, Points as HvPoints,
+    ImageStack as HvImageStack, Nodes as HvNodes, Path as HvPath,
```

Comment:
Please doublecheck if this is ok

Reply:
ImageStack is supported on all the versions Geoviews supports.

Reply:
I guess not since it was released in 1.18.0

Reply:
How should we address it? Pin holoviews>=1.18?

Reply:
At the top of my head, in pseudo code. 

``` python
imports
...
try:
    from holoviews import ImageStack as HvImageStack
except ModuleNotFoundError:
    class HvImageStack:
         def __init__(self, *args, **kwargs):
               raise Error('ImageStack requires Holoviews 1.18 or greater')
```

Reply:
I had to check it in __init__ of ImageStack since it instantiates the other class first.
<img width="988" alt="image" src="https://github.com/holoviz/geoviews/assets/15331990/da29efdc-ed0c-4930-a418-7f274c6195b3">